media-gfx/yacreader/yacreader-9.0.0.recipe

```diff
@@ -4,18 +4,21 @@ and managing your digital comic collection."
 HOMEPAGE="http://www.yacreader.com/"
 COPYRIGHT="Luis Ángel San Martín Rodríguez"
```

Comment:
The copyright year should be provided. As the author don't seem to left this information easily accessible, assuming it's the year of the latest release should be fine

```diff
-COPYRIGHT="Luis Ángel San Martín Rodríguez"
+COPYRIGHT="2018 Luis Ángel San Martín Rodríguez"